src/PrestaShopBundle/Controller/Admin/CommonController.php

```diff
@@ -53,7 +53,7 @@ class CommonController extends FrameworkBundleAdminController
      * {% render controller('PrestaShopBundle\\Controller\\Admin\\CommonController::paginationAction',
      *   {'limit': limit, 'offset': offset, 'total': product_count, 'caller_parameters': pagination_parameters}) %}
      *
-     * @Template
+     * @Template("@PrestaShop/Admin/Common/pagination.html.twig")
```

Comment:
We can only override templates using "arobased" notation.

Reply:
no you are able to overwrite all templates, but if you are allowed for breaking changes just do it in favor of best practices

Reply:
Hello Daniel, I've tried with the alternative syntax and it doesn't work Twig is unable to find the overriden template.

Reply:
for a clean overwrite you should implemented some some custom Twig loader / resolver as mentioned before about the activeModules.

another solution would be to load active without container access. its also done by Shopware https://github.com/shopware/shopware/blob/5.3/engine/Shopware/Kernel.php#L452

Reply:
Yes, for the templates I'm good thanks to a custom Twig loader as you suggested (thanks again). I'm looking at Shopware code  :)